apps/src/calc/calc.js

```diff
@@ -80,6 +81,12 @@ function getTokenList(one, two) {
   if (two instanceof Equation) {
     two = two.expression;
   }
+  if (jsnums.isSchemeNumber(one)) {
+    one = new ExpressionNode(one);
+  }
+  if (jsnums.isSchemeNumber(two)) {
+    two = new ExpressionNode(two);
+  }
   if (typeof(one) === 'string') {
```

Comment:
In this particular context, marked only indicates that the two are different. I could probably make that clearer by changing the name as you suggest.